debezium-platform-conductor/pom.xml

```diff
@@ -162,6 +164,22 @@
                 <version>${mockwebserver.version}</version>
             </dependency>
 
+            <!--Temporary forced since newer version are not compatible with current kubernetes client -->
```

Comment:
Is this necessary? I'd suppose that overring of properties should be enough? Also the comment should be there.